post-install.txt

```diff
@@ -12,13 +12,17 @@ chroot /rootfs /usr/bin/git clone -b master https://github.com/openhab/openhabia
 if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
 ln -s /opt/openhabian/openhabian-setup.sh /rootfs/usr/local/bin/openhabian-config
 
-echo "[openHABian] === Start executing 'openhabian-setup.sh' ==="
+echo -n "[openHABian] Backup installation configuration..."
+cp /rootfs/boot/installer-config.txt /rootfs/opt/openhabian/installer-config.txt.netinstalled
```

Comment:
Thinking one step further: You need to create this copy outside the openhabian folder. This folder is git controlled and cleaned on every `sudo openhabian-config` -> Update.

I would also name the file differently. How about `/opt/openhabian.conf` or even `/etc/openhabian.conf` I would not use two different filenames. Just store whatever information you need inside the config, that's what it is for.